lib/common/ListTile.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+
+class ListTileDisplay extends StatelessWidget {
+  final Widget _widget;
+  ListTileDisplay(this._widget);
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        children: <Widget>[
+          Container(
+            height: 50,
+            decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(10),
+            ),
+            margin: EdgeInsets.fromLTRB(30, 0, 30, 0),
```

Comment:
You can use EdgeInsets.symmetric as well here as you are only giving margin horizontally and equally on both sides